x-pack/solutions/chat/plugins/workchat-app/server/services/orchestration/agent_graph.ts

```diff
@@ -48,14 +48,15 @@ export const createAgentGraph = async ({
 
   const toolsProvider = new ToolsProvider({ session, logger });
 
+  // @ts-ignore
```

Comment:
can we be more specific here, either add specific rule that it ignores, or if we need to ignore all at least add comment why ;)

Reply:
just deleted the function, since it's not used.